core/config.go

```diff
@@ -5,12 +5,31 @@ import (
 	"errors"
 	"fmt"
 
+	retry "github.com/avast/retry-go"
 	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/gogoproto/proto"
 	"github.com/hyperledger-labs/yui-relayer/log"
 	"github.com/hyperledger-labs/yui-relayer/utils"
 )
 
+var config ConfigI
+
+type PathEndName string
```

Comment:
"path end" has no name.
`ConfigIDType` or `PathIDType` or something is easy to understand.

Reply:
Change to `ConfigIDType`